packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -296,7 +305,10 @@ class DayPicker extends StatelessWidget {
     final List<Widget> result = <Widget>[];
     for (int i = localizations.firstDayOfWeekIndex; true; i = (i + 1) % 7) {
       final String weekday = localizations.narrowWeekdays[i];
-      result.add(new Center(child: new Text(weekday, style: headerStyle)));
+      result.add(new ExcludeSemantics(
+        excluding: true,
```

Comment:
nit: `excluding: true` is the default, you could remove this line (here and in a couple of more places below).

Reply:
Done.